src/ontology/cl-edit.owl

```diff
@@ -17034,21 +17034,28 @@ AnnotationAssertion(rdfs:label obo:CL_0002324 "myoepithelial cell of mammary gla
 EquivalentClasses(obo:CL_0002324 ObjectIntersectionOf(obo:CL_0000185 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001911)))
 SubClassOf(obo:CL_0002324 obo:CL_0000185)
 
-# Class: obo:CL_0002325 (mammary alveolar cell)
+# Class: obo:CL_0002325 (lactocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:19022771") obo:IAO_0000115 obo:CL_0002325 "The milk-producing cell of the alveolar unit that emerges during pregnancy.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:19022771") Annotation(oboInOwl:hasDbXref "PMID:28717367") Annotation(oboInOwl:hasDbXref "PMID:35091553") Annotation(oboInOwl:hasDbXref "PMID:35377806") Annotation(oboInOwl:hasDbXref "doi:10.1177/156482659601700404") obo:IAO_0000115 obo:CL_0002325 "A glandular epithelial cell that is part of a mammary gland alveolus and differentiates from luminal adaptive secretory precursor cells during lactogenesis I. Following secretory activation during lactogenesis II, a lactocyte is involved in the biosynthesis and/or concentration of milk oligosaccharides, lactose, micronutrients, fat, hormones, immunoglobulins, and cytokines, as well as their transport into the lumen of the lactating mammary gland.")
```

Comment:
- Consider noting milk-production in the first sentence.
- "... differentiates from luminal adaptive secretory precursor cells." Consider making cells singular.